src/main/java/org/shanerx/tradeshop/Utils.java

```diff
@@ -724,6 +724,79 @@ public Block findShopChest(Block sign) {
     }
 
     /**
+     * Sets the name of the inventory
+     *
+     * @param state   blockState to change the name of
+     * @param name    original name of inventory, null to use generic name
+     * @param owners  List of inventory owners
+     * @param members List of inventory members
+     * @return void
+     */
+    public void changeInvName(BlockState state, String name, List<OfflinePlayer> owners, List<OfflinePlayer> members) {
+        StringBuilder sb = new StringBuilder();
+        if (name.equalsIgnoreCase("") || name == null) {
```

Comment:
The null check should precede `.equalsIgnoreCase`

Reply:
I can change both of those